docs/install.md

```diff
@@ -1,159 +1,162 @@
 # Installing Tekton Pipelines
 
-Use this page to add the component to an existing Kubernetes cluster.
+This guide explains how to install Tekton Pipelines. It covers the following topics:
 
-* [Pre-requisites](#pre-requisites)
-* [Versions](#versions)
-* [Installing Tekton Pipelines](#installing-tekton-pipelines)
+* [Before you begin](#before-you-begin)
+* [Installing Tekton Pipelines on Kubernetes](#installing-tekton-pipelines-on-kubernetes)
 * [Installing Tekton PIpelines on OpenShift/MiniShift](#installing-tekton-pipelines-on-openshiftminishift)
```

Comment:
Nit of thing you didn't change :angel: : `PIpelines` -> `Pipelines` (lowercase i)

Reply:
Hey, that's how I make sure reviewers actually read the doc... just kidding! Thank you, I often miss small things like this after a whole day of writing. Fixed. :)